packages/manager/apps/web/client/app/hosting/advices/hosting-pro/advice-hosting-pro.controller.js

```diff
@@ -0,0 +1,148 @@
+import compact from 'lodash/compact';
+import endsWith from 'lodash/endsWith';
+import filter from 'lodash/filter';
+import get from 'lodash/get';
+
+export default class {
+  /* @ngInject */
+  constructor(
+    $http,
+    $q,
+    $state,
```

Comment:
Your component is not related to routing so it shouldn't use `$state` or `$stateParams`.

Reply:
This is removed :)